core/trino-main/src/main/java/io/trino/execution/AddColumnTask.java

```diff
@@ -234,4 +256,13 @@ private Type getSupportedType(Session session, CatalogHandle catalogHandle, Map<
                 .getSupportedType(session, catalogHandle, tableProperties, type)
                 .orElse(type);
     }
+
+    private static io.trino.spi.connector.ColumnPosition toConnectorColumnPosition(ColumnPosition columnPosition)
+    {
+        return switch (columnPosition) {
+            case FIRST -> io.trino.spi.connector.ColumnPosition.FIRST;
+            case AFTER -> io.trino.spi.connector.ColumnPosition.AFTER;
+            case LAST -> io.trino.spi.connector.ColumnPosition.LAST;
```

Comment:
LAST is missing in the grammar. It should be allowed in the grammar, and should be the default if it's omitted.